java/client/src/main/java/com/vesoft/nebula/client/graph/meta/MetaInfo.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.client.graph.meta;
+
+import com.google.common.collect.Maps;
+import com.google.common.net.HostAndPort;
+import com.vesoft.nebula.meta.EdgeItem;
+import com.vesoft.nebula.meta.TagItem;
+import java.util.List;
+import java.util.Map;
+
+public class MetaInfo {
+    private final Map<String, Integer> spaceNameMap = Maps.newHashMap();
+    private final Map<String, Map<Integer, List<HostAndPort>>>
+            spacePartLocation = Maps.newHashMap();
+    private final Map<String, Map<Long, TagItem>> spaceTagItems = Maps.newHashMap();
+    private final Map<String, Map<Long, EdgeItem>> spaceEdgeItems = Maps.newHashMap();
+    private final Map<String, Map<String, Long>> tagNameMap = Maps.newHashMap();
+    private final Map<String, Map<Long, String>> tagIdMap = Maps.newHashMap();
+    private final Map<String, Map<String, Long>> edgeNameMap = Maps.newHashMap();
+    private final Map<String, Map<Long, String>> edgeIdMap = Maps.newHashMap();
+    private final Map<String, Map<Integer, HostAndPort>> leaders = Maps.newHashMap();
+
+    protected int connectionRetry;
+    protected int connectionTimeout;
+    protected int executionRetry;
+    protected int timeout;
+
+
+    public MetaInfo() {
```

Comment:
how to initial meta info ?

Reply:
When  meta client is created, metaInfo is create at the same time.

Reply:
But the Constructor is empty...

Reply:
Because no variable need to be assigned. All variables are assigned when MetaInfo is created.
When meta client is connected, the metaInfo will be put more values.

Reply:
how to modify the private variable?

Reply:
There are public get method for variables.

Reply:
why not support some method to update the data in maps?